### ISSUE TITLE
Family Tree

### ISSUE THREAD

**Issue by kevinlewis13** [kevinlewis13/harrys-chicken#12]
that could be neat to implement for like an about page or something. maybe hovering over pictures of people and getting a little blurb.

**Comment by andyjac** [comment]
yeah, maybe it's the first thing you see when you hit the site, that way we're clear about the different sides to the website (chicken vs. coffee)

**Comment by andyjac** [comment]
for this we'll need pictures of people.
